R (#498)
Codecademy/docs: content/r/r.md

---
Title: 'R'
Description: 'R is a popular open-source interpreted programming language used for statistical computing and graphics.'
Codecademy Hub Page: 'https://www.codecademy.com/catalog/language/r'
CatalogContent:
  - 'learn-r'
  - 'paths/computer-science'
---

> R is a popular open-source interpreted programming language used for statistical computing and graphics.
> It was created by Ross Ihaka and Robert Gentleman and version 1.0 was released in February 2000.

Reviewer: ```suggestion
R is a popular open-source interpreted programming language used for statistical computing and graphics. It was created by Ross Ihaka and Robert Gentleman and version 1.0 was released in February 2000.
```

Since the target of both sentences appear related, let's keep them together rather than on separate lines.

> R supports procedural programming and object-oriented programming and its functionality can be easily expanded through functions and extensions.
> A variety of packages for R are available through the R community.
> It is widely used by statisticians and data scientists for statistics, data analysis and data visualization,
> and it's an important tool for people planning to enter these fields.

Reviewer: ```suggestion
R supports procedural programming and object-oriented programming and its functionality can be easily expanded through functions and extensions.

A variety of packages for R are available through the R community. They are widely used by statisticians and data scientists for tasks such as statistics, data analysis, and data visualization.
```

Reviewer: @SSwiniarski Should we also include a section for installing R?

Author: @Dusch4593, I don't know. Do any other main language entries go into the details of installation? Should there be separate "installation" entries for various languages?
